src/flexible_type/flexible_type_converter.hpp

```diff
@@ -128,9 +128,54 @@ template <> struct ft_converter<CVTR__FLEXIBLE_TYPE_EXACT> {
     dest = src;
   }
  
+  static void get(flex_float& dest, const flexible_type& src) {
```

Comment:
What's with these static void methods? I don't see any calls to them in this commit (and as of now what I'm looking at is the latest commit in this PR). Were calls to these methods going to come later?

Reply:
No, they are being used by the template calling classes at the end of this file.  

The way this works is that it tests every one of these classes in order by calling the matches() static method, and if it is true, it calls the get / set methods as appropriate. 

Because of how the templating works, none of these classes are ever instantiated, hence the static method.  Also, the destination and source of the conversions need to be overloaded by type, they are arguments, hence the void.